packages/jest-circus-allure-reporter/src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './reporter';
+
+export * from './types';
+
+export { AllureRuntime, AllureConfig } from 'allure-js-commons';
```

Comment:
This is to eliminate the need to have `allure-js-commons` as a direct dependency of `@automattic/calypso-e2e`. 

These exports are used in `environment.ts` during the initialization of Allure reporter.